docs/getting-started/upgrade-to-v4.md

```diff
@@ -205,6 +81,53 @@ export default function App({
 
 Introduced in https://github.com/nextauthjs/next-auth/releases/tag/v4.0.0-next.12
 
+## Providers
+
+OAuth providers now need to be imported individually.
```

Comment:
```suggestion
Providers now need to be imported individually.
```

The main motivation for this is btw that someone not using the `Email` provider, won't have to pull in `nodemailer`. Even after we made it a peer dependency, with the old import, it would through an error, `nodemailer` not being found.

We might do something like `import {...} from next-auth/providers/oauth`, I don't know. but for this suggestion, the important part is that it's about all providers, not just OAuth ones.